### PR TITLE
[WIP] Clean up config

### PR DESCRIPTION
**Pull request by picklelo**
Clean up config variables and CLI commands.

### CODE REVIEW

File: reflex/reflex.py [hunk 1]
```diff
@@ -234,50 +187,76 @@ def export(
...
-        zip=zipping,
+        backend=export_backend,
+        frontend=export_frontend,
         deploy_url=config.deploy_url,
     )
 
     # Post a telemetry event.
     telemetry.send("export", config.telemetry_enabled)
+    console.success(
+        """App exported to[bold]backend.zip[/bold] and [bold]frontend.zip[bold]."""
```
Comment (masenf) [inline reflex/reflex.py:216]: i think we should keep track of output files and not say we exported a particular file if we didn't (#1285)

```python
outputs = set()
if export_backend:
    if zipping:
        outputs.add("backend.zip")
    else:
        outputs.add("app")
if export_frontend:
    if zipping:
        outputs.add("frontend.zip")
    else:
        outputs.add(".web/_static")
```
...
```python
console.success(
    "App exported to " + " and ".join(f"[bold]{filename}[/bold]" for filename in outputs)
)
```

File: reflex/reflex.py [hunk 2]
```diff
@@ -168,63 +155,29 @@ def run(
...
-        httpx.put(frontend, data=f)  # type: ignore
-
-    with open(constants.BACKEND_ZIP, "rb") as f:
-        httpx.put(backend, data=f)  # type: ignore
-
-
 @cli.command()
 def export(
-    zipping: bool = typer.Option(
-        True, "--no-zip", help="Disable zip for backend and frontend exports."
```
Comment (masenf) [inline reflex/reflex.py:221]: is there a reason `--no-zip` is going away? it's very convenient for docker container builds

File: reflex/reflex.py (hunk 1 above)
Comment (picklelo) [reply reflex/reflex.py:216]: Good call, will change this

File: reflex/reflex.py (hunk 2 above)
Comment (picklelo) [reply reflex/reflex.py:221]: I'll add this back

**Comment by masenf** [comment]
Fixed via #1591

**Pull request closed without merging**